bridge.js

```
@@ -12,7 +13,25 @@ const hubName = process.env.TTN_AZURE_HUBNAME;
 const keyName = process.env.TTN_AZURE_KEYNAME;
 const key = process.env.TTN_AZURE_KEY;
 
-const bridge = new ttnazureiot.Bridge(region, appId, accessKey, hubName, keyName, key);
+var options;
+
+try {
+  const mqttCertPath = process.env.TTN_MQTT_CERT;
```

Fallback to hardcoded path; ` || '/etc/ttn/mqtt.ca';`